### PR TITLE
Added option to use Breeze Dark only if the client's dark mode is ena…

### PR DESCRIPTION
**Pull request by untbu**
Added option to make Breeze Dark dependent on prefers-color-scheme css media query

### CODE REVIEW

File: lib/Settings/Admin.php [hunk 1]
```diff
@@ -61,10 +61,12 @@ public function getForm(): TemplateResponse
     {
         $themeEnabled = $this->config->getAppValue($this->appName, 'theme_enabled', "0");
         $themeLoginPage = $this->config->getAppValue($this->appName, 'theme_login_page', "1");
+        $themeAutomaticActivation = $this->config->getAppValue($this->appName, 'theme_automatic_activation_enabled', "1");
```
Comment (mwalbeck) [inline lib/Settings/Admin.php:64]: The default value here should be "0" instead of "1"

File: lib/Settings/Personal.php [hunk 2]
```diff
@@ -73,9 +73,12 @@ public function __construct(
     public function getForm(): TemplateResponse
     {
         $default = $this->config->getAppValue($this->appName, 'theme_enabled', "0");
+        $defaultAutomaticActivation = $this->config->getAppValue($this->appName, 'theme_automatic_activation_enabled', "1");
```
Comment (mwalbeck) [inline lib/Settings/Personal.php:76]: The default value should also be "0" here

File: lib/Settings/Personal.php [hunk 3]
```diff
@@ -73,9 +73,12 @@ public function __construct(
...
         $themeEnabled = $this->config->getUserValue($this->userId, $this->appName, 'theme_enabled', $default);
+        $themeAutomaticActivation = $this->config->getUserValue($this->userId, $this->appName, 'theme_automatic_activation_enabled', $default);
```
Comment (mwalbeck) [inline lib/Settings/Personal.php:78]: Looks like you forgot to replace `$default` with `$defaultAutomaticActivation`

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Added
+
+-   Add option to use Breeze Dark only if the client's dark mode is enabled
```
Comment (mwalbeck) [inline CHANGELOG.md:7]: Could you add the following to the changelog line, like it's done on previous entries

`[#98](https://github.com/mwalbeck/nextcloud-breeze-dark/issues/98)`

**Review by mwalbeck — CHANGES_REQUESTED** [review]
Thank you very much for the contribution! From my testing, it works quite well! I did find a couple minor things, but overall I'm quite happy with it.

If you could fix the mentioned issues and squash the pr into one commit, then I think it's ready to merge!

**Comment by untbu** [comment]
Sorry, but i don't know how to squash the commits. When i do it locally, i get an error while pushing it and on github i dont' even see the commits in my fork. Don't you have an option to squash and merge?

**Review by mwalbeck — APPROVED** [review]

**Comment by mwalbeck** [comment]
No worries! I generally prefer to let the committer do the squashing and commit message, but it's no more than a preference.

Thank you again for the contribution! I greatly appreciate it!

Cheers!

**Pull request merged**